### PR TITLE
Update data-classification-activity-explorer.md

### PR DESCRIPTION
**Pull request by shadansan**
@chrfox  I updated this with recent update we made for security reader role. Please kindly help with the next step!

### CODE REVIEW

File: microsoft-365/compliance/data-classification-activity-explorer.md [hunk 1]
```diff
@@ -94,6 +94,7 @@ Here's a list of applicable role groups that you can use. To learn more about th
 - Security administrator
 - Security Reader
 
+Note that security reader role can only view activities and is not allowed to view the sensitive content.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/data-classification-activity-explorer.md:97]:
```suggestion
Note that the Security Reader role can only view activities and is not allowed to view sensitive content.

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @shadansan, kindly commit the suggested changes. Many thanks.

**Comment by shadansan** [comment]
@***@***.***> how do I commit the changes? Can we publish this?
Best regards
Shadan

**Comment by JohanFreelancer9** [comment]
@shadansan, to commit a suggested change in GitHub, navigate to the suggested change you'd like to commit. To apply the change in its own commit, click **Commit suggestion**. Many thanks.

**Comment by shadansan** [comment]
Thank you @***@***.***> engineering asked me to make another update. So I made another update and close the old PR. Can you please check the new one. I mentioned you there.
Best regards
Shadan

**Comment by kmkoenen** [comment]
Shandan make a couple of related changes. Please republish.

**Comment by kmkoenen** [comment]
Shandan made a couple of changes requested by the engineering team. Please republish.

**Comment by Cee3097** [comment]
Publish update for 365 approved only for event official peak time

**Comment by kmkoenen** [comment]
I'm new. Please clarify this comment"Publish update for 365 approved only for event official peak time"
Thanks

**Comment by Cee3097** [comment]
I'm to damn old and experienced to have to explain my ass today. Update
(PR#10969)

**Comment by dstrome** [comment]
@Cee3097 - Profanity is against the [Microsoft Code of Conduct](https://opensource.microsoft.com/codeofconduct/).

**Pull request closed without merging**